programs/bench.h

```diff
@@ -19,46 +19,135 @@ extern "C" {
 #define ZSTD_STATIC_LINKING_ONLY   /* ZSTD_compressionParameters */
 #include "zstd.h"     /* ZSTD_compressionParameters */
 
+#define BMK_COMPRESS_ONLY 2
+#define BMK_DECODE_ONLY 1
```

Comment:
If you need to switch between 2 modes depending on a parameter, prefer an `enum` type.